src/opnsense/mvc/app/controllers/OPNsense/Dnsmasq/forms/general.xml

```diff
@@ -100,6 +100,12 @@
         <type>checkbox</type>
         <help>If this option is set, we will not forward A or AAAA queries for plain names, without dots or domain parts, to upstream name servers. If the name is not known from /etc/hosts or DHCP then a "not found" answer is returned.</help>
     </field>
+    <field>
+        <id>dnsmasq.no_resolv</id>
+        <label>Do not forward to system defined DNS servers</label>
+        <type>checkbox</type>
+        <help>If this option is set, do not use (System: General Setup: DNS Servers). Only use upstream servers defined in (Services: Dnsmasq DNS & DHCP: Domains). This option is recommended when Unbound forwards local domain queries to Dnsmasq, so that all queries terminate without further lookups if they are unknown.</help>
```

Comment:
```suggestion
        <help>If this option is set, DNS forwarding to system nameservers (defined in System: General Setup: DNS Servers) will be disabled. Upstream servers defined in Services: Dnsmasq DNS & DHCP: Domains will still be used. This option is recommended when Unbound forwards local domain queries to Dnsmasq, so that all queries terminate without further lookups if they are unknown.</help>
```